docs/src/source/api/api_groups/jpeg2000/getStatus.rst

```diff
@@ -47,3 +47,16 @@ Example: string (JSON)
         "measurement": "EUVI-A 171"
       }
     }
+    
+        .. table:: Response Description
+    +-----------+----------+--------------+-------------------------------------------------------------------------------------------------+-----+
+    | Parameter     | Required |     Type     |                                           Description                                           |     |
+    +===========+==========+==============+=================================================================================================+=====+
+    | time          | Required | datetime     | The timestamp of the last available measurement.                                                |     |
+    +-----------+----------+--------------+-------------------------------------------------------------------------------------------------+-----+
+    | level         | Required | Integer      | The status level of the image based on time since it was captured.                              |     |

```

Comment:
I had to lookup what this meant. Please update the description to:

"Numeric indicator of how up-to-date the source is relative to normal operation. 1 = Helioviewer is showing the latest data, 5 = new data may be available, but hasn't reached helioviewer yet.